client/src/templates/Challenges/classic/editor.tsx

```diff
@@ -1277,6 +1277,18 @@ const Editor = (props: EditorProps): JSX.Element => {
 
   const firstFailedTest = props.tests.find(test => !!test.err);
 
+  const handleSubmitAndGoButtonBoolean = () => {
+    const canShowModal = sessionStorage.getItem('canOpenModal');
+
+    if (canShowModal === 'false' && process.env.NODE_ENV === 'development') {
+      return false;
+    } else {
+      challengeIsComplete();
+    }
+
+    return challengeIsComplete();
```

Comment:
```suggestion
    if (canShowModal === 'false' && process.env.NODE_ENV === 'development') {
      return false;
    }
    return challengeIsComplete();
```

We don't need an else statement when we're using an early return.

Reply:
Yeah this seems better will change it to that

Reply:
Ok done